src/ThirdParty/Gitlab/Adapter/GitLabIssueTracker.php

```diff
@@ -190,7 +196,18 @@ public function getComments($id)
             $this->client->api('issues')->show($this->getCurrentProject()->id, $id)
         );
 
-        return $issue->showComments();
+        $comments = [];
+        $issue->showComments();
```

Comment:
Ok you are right, the `setData` method is used in the `hydrate` one so we are blocked by the property check...

We now need to wait for the php-gitlab-api update to be published to simplify our code...